Xamarin.Android.Lite.Tests/Data/AndroidManifest_vNext.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="1" android:versionName="1.0" package="com.xamarin.android.lite" >
+    <uses-sdk android:minSdkVersion="19" android:targetSdkVersion="27" />
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+    <uses-permission android:name="android.permission.BATTERY_STATS" />
+    <uses-permission android:name="android.permission.CAMERA" />
+    <uses-permission android:name="android.permission.FLASHLIGHT" />
+    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
+    <uses-permission android:name="android.permission.VIBRATE" />
+    <uses-feature android:name="android.hardware.location" android:required="false" />
+    <uses-feature android:name="android.hardware.location.gps" android:required="false" />
+    <uses-feature android:name="android.hardware.location.network" android:required="false" />
+    <application android:label="Xamarin.Android.Lite" android:icon="@mipmap/icon" android:name="android.app.Application" android:debuggable="true" android:allowBackup="true">
+        <meta-data android:name="Xamarin.Android.Lite.Application" android:value="Xamarin.Android.Lite.Sample.App, Xamarin.Android.Lite.Sample" />
+        <activity android:theme="@style/MainTheme" android:label="Xamarin.Android.Lite" android:icon="@mipmap/icon" android:name="md5bff8b7c7908ce4fe5d805acf2300a9b4.MainActivity" android:configChanges="orientation|screenSize">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+                <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
+        </activity>
+        <service android:name="md5dcb6eccdc824e0677ffae8ccdde42930.KeepAliveService" />
+        <receiver android:label="Essentials Battery Broadcast Receiver" android:name="md5d630c3d3bfb5f5558520331566132d97.BatteryBroadcastReceiver" android:enabled="true" android:exported="false" />
+        <receiver android:label="Essentials Connectivity Broadcast Receiver" android:name="md5d630c3d3bfb5f5558520331566132d97.ConnectivityBroadcastReceiver" android:enabled="true" android:exported="false" />
+        <receiver android:label="Essentials Power Broadcast Receiver" android:name="md5d630c3d3bfb5f5558520331566132d97.PowerBroadcastReceiver" android:enabled="true" android:exported="false" />
+        <receiver android:name="md51558244f76c53b6aeda52c8a337f2c37.PowerSaveModeBroadcastReceiver" android:enabled="true" android:exported="false" />
```

Comment:
@jamesmontemagno or @redth

It's weird this manifest line got added in Xamarin.Essentials 0.8.0->0.9.1, but has no label.

Reply:
Hmmmmmmm I am not sure it is from us: https://github.com/xamarin/Essentials/search?q=broadcastreceiver&unscoped_q=broadcastreceiver

Reply:
Err.... It must be a bug in Xamarin.Android itself.

If I look at `obj\Debug\android\AndroidManifest.xml`:
```
<receiver android:enabled="true" android:exported="false" android:name="md51558244f76c53b6aeda52c8a337f2c37.PowerSaveModeBroadcastReceiver" />
```
(This is from Xamarin.Android.Lite -- the regular Xamarin.Android project I'm using for the prebuilt APK)

But if I decompile Xamarin.Essentials from either `obj\Debug\android\assets` or `%userprofile%\.nuget\packages\xamarin.essentials`, I don't even see a `PowerSaveModeBroadcastReceiver` type.

I'll look into this more next week to make sure it's not a bug.

Reply:
Ok, this type got added to Xamarin.Forms: https://github.com/xamarin/Xamarin.Forms/blob/ffa53d31137b61e34192521cc10aae5c151d0c98/Xamarin.Forms.Platform.Android/PowerSaveModeBroadcastReceiver.cs

Not sure what it is for...

Reply:
Oh interesting.... yeah, they should add a name. I think they did this because animations were causing issues when in power save mode.